02 - Community Expansions/02.01 Plugins by Category/Search and query plugins.md

```diff
@@ -23,14 +23,13 @@ Querying
 - [[obsidian-query-language|Obsidian Query Language]]
 - [[dataview|Dataview]]
 - [[obsidian-spotlight|Spotlight]]
-- [[obsidian-dice-roller|Dice Roller]]
 - [[obsidian-related-notes-finder|Related Notes Finder]]
-- [[taskbone-ocr-plugin|Taskbone OCR]]
 - [[obsidian-relative-find|Relative Find]]
 - [[obsidian-apply-patterns|Apply Patterns]]
```

Comment:
this one can't be used for searching though, it's for applying a replace-regex to note